setup.py

```diff
@@ -8,6 +12,93 @@ def get_advanced_templates():
     return [template_base + name + '.json' for name in template_names]
 
 
+def get_packages():
+    if is_windows:
+        return [
+            'dcos_installer',
+            'gen',
+            'gen.build_deploy',
+            'pkgpanda',
+            'pkgpanda.build',
+            'pkgpanda.http',
+            'release',
+            'release.storage']
+    else:
+        return [
+            'dcos_installer',
+            'gen',
+            'gen.build_deploy',
+            'pkgpanda',
+            'pkgpanda.build',
+            'pkgpanda.http',
+            'release',
+            'release.storage',
+            'ssh']
```

Comment:
Why are we duplicating all of these module names when `ssh` is the only difference between them?

Reply:
ok